### PR TITLE
csrf issue fixed

### LINKED ISSUE #1098: Check CSRF token security throughout system

### PR DESCRIPTION
**Pull request by ashu555**
Fixes #1098

### CODE REVIEW

File: public/common/js/2fa.js [hunk 1]
```diff
@@ -105,7 +105,7 @@
                                                     $("#pass_btn").html("<i class='fa fa-circle-o-notch fa-spin fa-1x fa-fw'></i>Verifying...");
                                                     $.ajax({
                                                         url : "2fa/setupValidate",
-                                                        method : 'POST',
+                                                        method : 'GET',
```
Comment (avinash403) [inline public/common/js/2fa.js:108]: why is it changed to get?

File: resources/views/themes/default1/layouts/master.blade.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <head>
         <meta charset="UTF-8">
         <title>@yield('title') | {{$set->favicon_title}}</title>
+        <meta name="csrf-token" content="{{ csrf_token() }}" />
```
Comment (avinash403) [inline resources/views/themes/default1/layouts/master.blade.php:10]: is this master in use?

Comment (ashu555) [reply resources/views/themes/default1/layouts/master.blade.php:10]: In admin panel

File: public/common/js/2fa.js (hunk 1 above)
Comment (ashu555) [reply public/common/js/2fa.js]: By mistake. was checking something. Fixed

**Comment by ashu555** [comment]
@SarfrazS  edge browser issue exists in development branch also. Create a seperate issue for this and assign to me

**Comment by ashu555** [comment]
> In the installer please correct the alert message. see the below is showing now.
> image
> 
> Note:- instead of showing click here to exit. please can we show click here to login

Fixed

**Comment by ashu555** [comment]
> In the environment step:- AAP URL field if we entered the only https://ladybirdweb.ml and click on install
> Migration table created successfully but when we click on click here to exit. Then throwing below error
> (same error getting when we installing on server and app url entered localhot).
> 
> appurl

Fixed

**Comment by avinash403** [comment]
@ashu555 I remember asking you to get it tested from at least 2 testers

**Pull request merged**